substratevm/src/com.oracle.svm.test/src/hello/#Hello.java#

```diff
@@ -0,0 +1,247 @@
+/*
+ * Copyright (c) 2020, 2020, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2020, 2020, Red Hat Inc. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package hello;
+
+// Checkstyle: stop
+
+import com.oracle.svm.core.AlwaysInline;
+import com.oracle.svm.core.NeverInline;
+
+import org.graalvm.word.PointerBase;
+import org.graalvm.word.WordFactory;
+import org.graalvm.word.UnsignedWord;
+import org.graalvm.word.SignedWord;
+
+import org.graalvm.nativeimage.UnmanagedMemory;
+
+public class Hello {
```

Comment:
Where is this class/file used?

Reply:
This is left over from an experiment. UnmanagedMemory is not used by Hello. Removed.